index.html

```diff
@@ -359,28 +359,17 @@ <h3>Proof Representations</h3>
           <h4>DataIntegrityProof</h4>
 
           <p>
-The `verificationMethod` property of the proof MUST be a URL.
-Dereferencing the `verificationMethod` MUST result in an object
-containing a `type` property with the value set to
-`Multikey`.
+A proof contains the attributes specified in the
+<a href="https://www.w3.org/TR/vc-data-integrity/#proofs">Proofs section</a>
+of [[VC-DATA-INTEGRITY]] with the following restrictions.
           </p>
-
           <p>
-The `type` property of the proof MUST be `DataIntegrityProof`.
+The `type` property MUST be `DataIntegrityProof`.
           </p>
           <p>
 The `cryptosuite` property of the proof MUST be `eddsa-rdfc-2022` or `eddsa-jcs-2022`.
           </p>
           <p>
-The `created` property of the proof MUST be an [[XMLSCHEMA11-2]]
-formatted date string.
```

Comment:
This doesn't make the `created` property optional; it drops all reference to it. I would expect something like this --
```suggestion
The `created` property of the proof is OPTIONAL; if present, it MUST be an
[[XMLSCHEMA11-2]] formatted date string.
```

Reply:
I think it's fine to remove it... the spec text states that the base requirements come from DI, and the requirements listed here are ADDITIONAL requirements.

Reply:
OK; I overlooked that this was replicating DI.

Reply:
(But I submit that the title of this PR should be changed accordingly.)

Reply:
Done (title of PR updated).